src/views/Home/components/DaysLeft/index.jsx

```diff
@@ -1,11 +1,11 @@
 import { useEffect, useState } from 'react';
-import { Doughnut } from 'react-chartjs-2';
-import GordonLoader from 'components/Loader';
 import session from 'services/session';
 import styles from '../Doughnut.module.css';
-import { Card, CardContent, CardHeader, Grid, Typography } from '@mui/material';
+import { Card, CardContent, CardHeader, Grid, Typography, checkboxClasses } from '@mui/material';
 import { theme360 } from 'theme';
 import { format, parseISO } from 'date-fns';
+import LinearProgress from '@material-ui/core/LinearProgress';
+import { withStyles } from '@material-ui/core/styles';
```

Comment:
```suggestion
import LinearProgress from '@material-ui/core/LinearProgress';
import { withStyles } from '@material-ui/core/styles';
```
pretty sure we can do these from scss files